tikv/txn.go

```diff
@@ -175,6 +175,18 @@ func (txn *KVTxn) Get(ctx context.Context, k []byte) ([]byte, error) {
 	return ret, nil
 }
 
+// BatchGet gets kv from the memory buffer of statement and transaction, and the kv storage.
+// Do not use len(value) == 0 or value == nil to represent non-exist.
+// If a key doesn't exist, there shouldn't be any corresponding entry in the result map.
+func (txn *KVTxn) BatchGet(ctx context.Context, keys [][]byte) (map[string][]byte, error) {
+	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
```

Comment:
It seems that other methods don't use opentracing. I think we can remove it and only record it in tidb side.

Reply:
@disksing done!